threatingestor/config.py

```diff
@@ -100,8 +100,9 @@ def operators(self):
                         # pass in special filter_string option
                         kwargs['filter_string'] = value
 
-                    elif key == ALLOWED_SOURCES:
-                        kwargs[key] = [s.strip() for s in value]
```

Comment:
I removed this because we are using YAML syntax here instead of the CSV we used previously, so there's no need to manually strip whitespace off the parsed strings.